docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,36 +1,150 @@
 # Parameters
 
-The ROS packages in Autoware have ROS parameters. You need to customize the parameters depending on your applications.
-It is recommended not to set default values when declaring ROS parameters to avoid unintended behaviors due to accidental use of default values.
-Instead, set parameters from configuration files named `*.param.yaml`.
+Autoware ROS nodes have declared parameters which values are provided during the node start up in the form of a parameter file. All the expected parameters with corresponding values should exist in the parameter file. Depending on the application, the parameter values might need to be modified.
 
-For understanding ROS 2 parameters, also check out the official documentation [Understanding parameters](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Parameters/Understanding-ROS2-Parameters.html).
+Find more information on parameters from the official ROS documentation:
 
-## Parameter files
+- [Understanding ROS 2 Parameters](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Parameters/Understanding-ROS2-Parameters.html)
+- [About ROS 2 Parameters](https://docs.ros.org/en/humble/Concepts/About-ROS-2-Parameters.html)
 
-Autoware has the following two types of parameter files for ROS packages:
+## JSON Schema
```

Comment:
As an developer who have not followed the AWF DevOps discussions, here are some high level comments on this modification:

1. Could you elaborate more on the background of this documentation? As a beginner, I could not get the whole idea until following the latest thread in this PR (especially [the comment by mitsudome-san](https://github.com/autowarefoundation/autoware-documentation/pull/367/files#r1205552883)). Maybe adding some phrase like "to have a single source of truth" helps.
2. Putting this section at the very top of this page is confusing. Consider moving this section to the last of this page or creating a separate page for this.

Comments are also welcome from @kaspermeck-arm and others as well!